crates/sui-graphql-rpc/src/context_data/db_data_provider.rs

```diff
@@ -490,9 +495,16 @@ impl PgManager {
         last: Option<u64>,
         before: Option<String>,
         filter: Option<ObjectFilter>,
+        name_service_name: bool,
     ) -> Result<Option<(Vec<StoredObject>, bool)>, Error> {
         let mut query = objects::dsl::objects.into_boxed();
 
+        if name_service_name {
+            query = query.filter(
+                objects::dsl::df_object_type.eq("0xd22b24490e0bae52676651b4f56660a5ff8022a2576e0089f79b3c88d44e08f0::name_record::NameRecord".to_string())
```

Comment:
Why is this hardcoded in this query?`0xd22b24490e0bae52676651b4f56660a5ff8022a2576e0089f79b3c88d44e08f0` If we want this to be usable on other networks (say testnet) we'll need to have the configurable with a default for mainnet

Reply:
Also it seems like this parameter is unused? why do we want to filter for this?

Reply:
I tried to do something slick w/ the Domain, NameRecord table but I don't think the approach works. Will remove this code as well